Framework/CurveFitting/test/Functions/ProcessBackgroundTest.h

```diff
@@ -339,35 +339,59 @@ class ProcessBackgroundTest : public CxxTest::TestSuite {
   }
 };
 
-class ProcessBackgroundTestPerformance : public CxxTest::TestSuite {
+// Performance test for "Delete Region" option
+class ProcessBackgroundDRTestPerformance : public CxxTest::TestSuite {
 public:
   // This pair of boilerplate methods prevent the suite being created statically
   // This means the constructor isn't called when running other tests
-  static ProcessBackgroundTestPerformance *createSuite() {
-    return new ProcessBackgroundTestPerformance();
+  static ProcessBackgroundDRTestPerformance *createSuite() {
+    return new ProcessBackgroundDRTestPerformance();
```

Comment:
I think that name won't work. `cxxtest` is quite stupid and will not pick up any tests apart from those named `ClassnameTest` and `ClassnameTestPerformance`. Or did this work for you?